src/Adapter/Module/ModuleDataProvider.php

```diff
@@ -133,4 +135,25 @@ protected function getModulesDir()
     {
         return _PS_MODULE_DIR_;
     }
+
+    /**
+     * Check if the module has been enabled on mobile
+     * @param string $name The technical module name to check
+     * @return int The devices enabled for this module
+     */
+    private function getDeviceStatus($name)
+    {
+        $id_shops = (new Context())->getContextListShopID();
+        // ToDo: Load list of all installed modules ?
```

Comment:
we may create a ticket on forge instead :)

https://www.youtube.com/watch?v=ZXsQAXx_ao0